package.json

```diff
@@ -45,7 +45,7 @@
     },
     "advancedLocalNodeModules": {
```

Comment:
Are you saying that I can use project manager to set advancedLocalNodeModules on a per-project basis?